Makefile

```diff
@@ -398,6 +398,18 @@ endef
 		make all install; \
 	fi
 
+
+HOOK_BIN_DIR ?= /usr/libexec/oci/hooks.d/
+HOOK_DIR ?= /usr/share/containers/oci/hooks.d
+
+generate-seccomp.install: 
+	install -m 755 bin/oci-trace-hook $(HOOK_BIN_DIR)
+	cp cmd/podman/oci-trace-hook/run.json $(HOOK_DIR)
+	cp cmd/podman/oci-trace-hook/stop.json $(HOOK_DIR)
+
+generate-seccomp: 
+	$(GO) build -o bin/oci-trace-hook $(PROJECT)/cmd/podman/oci-trace-hook 
```

Comment:
Note: we need to update the install instructions as well since building the hook requires `bcc-devel` to be installed.

Reply:
Still valid. @cevich can we add the `bcc-devel` pckage to our CI? We need it to build the new OCI hook.

Reply:
Yes, but I'm up to my eyeballs in rolling new VM images for various unrelated purposes.  So please feel free to attempt this yourself, here or in another PR.  The files that need changing are:

* `contrib/cirrus/packer/fedora_setup.sh`
* `contrib/cirrus/packer/ubuntu_setup.sh`

[Docs for testing those changes is here (add magic phrase to the PR (big) text-box)](https://github.com/containers/libpod/tree/master/contrib/cirrus#test_build_cache_images_task-task).  Assuming the images build, it will validate them against the integration tests automatically .  Assuming that passes,[ following the `manual steps` (and removing the magic string) ](https://github.com/containers/libpod/tree/master/contrib/cirrus#verify_test_built_images-task) will place the new images in use for everybody, after the PR merges.

@haircommander has done this a few times now, proof it's achievable by !@cevich :smile:

Reply:
Argh, dangit, more complicated...you're going to need to rebase onto #3632 as well (until it merges).

Reply:
Thanks a ton, @cevich!

@weirdwiz, after rebasing we need to add the package to https://github.com/containers/libpod/blob/master/contrib/cirrus/packer/ubuntu_setup.sh#L30 for Ubuntu and do the same for Fedora.